ui/src/components/sermons/OnlinePageButtons.js

```diff
@@ -12,22 +12,134 @@ import {
   useDisclosure,
 } from '@chakra-ui/react';
 
-const OnlinePageButtons = () => {
+const OnlinePageButtons = (props) => {
+  const { isIgnite } = props;
   const { isOpen, onOpen, onClose } = useDisclosure();
 
-  return (
-    <Stack direction={['column', 'row']}>
+  const IgniteConnectionButton = () => {
```

Comment:
![Uploading Screenshot 2022-05-21 at 21.54.50.png…]()
dont think that part is relevant/should be shown